libs/libGamedata/gameData/SelectionMapInputData.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2005 - 2023 Settlers Freaks (sf-team at siedler25.org)
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+#include <boost/filesystem.hpp>
+#include <string>
+#include <vector>
+
+struct ImageResource
+{
+    boost::filesystem::path filePath;
+    unsigned index;
+    ImageResource(boost::filesystem::path path = boost::filesystem::path(), unsigned index = 0)
+        : filePath(std::move(path)), index(index){};
+};
+
+struct MissionSelectionInfo
+{
+    unsigned maskAreaColor = 0;
+    Position ankerPos;
+};
+
+struct SelectionMapInputData
+{
+    SelectionMapInputData() = default;
+
+    // background image for the selection map
```

Comment:
To differentiate comments from docstrings the latter should use triple-slashes:
```suggestion
    /// background image for the selection map
```

Not so important though just mentioning it as those likely need to be touched anyway.